lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
+     * @return string
+     */
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).